tests/components/UserActions/UserActions.spec.ts

```diff
@@ -188,6 +188,49 @@ describe('UserActions', () => {
         });
     });
 
+    it('should pass custom init options to each of the sub components', () => {
+        const FAKE_USER_ID = 'someUserId' + Math.random();
+        const initOptions = {} as any;
```

Comment:
Why `as any` ?

Reply:
Without the `as any`, TypeScript was complaining when trying to execute `initOptions[component]` where `component` is of type String.

Reply:
Specified `initOptions` as a dictionary instead of `any`.